app/src/main/java/eu/kanade/presentation/track/TrackInfoDialogSelector.kt

```diff
@@ -123,7 +123,7 @@ fun TrackScoreSelector(
         content = {
             WheelTextPicker(
                 modifier = Modifier.align(Alignment.Center),
-                startIndex = selections.indexOf(selection).coerceAtLeast(0),
+                startIndex = if (selections.indexOf(selection) > 0) selections.indexOf(selection) else (selections.size / 2),
```

Comment:
Could probably do something like

```suggestion
                startIndex = selections.indexOf(selection).takeIf { it >= 0 } ?: selections.size / 2,
```

(Notably you need `>=`)

Reply:
Are you sure I need to use >=. In this case? It will only work for Anilist with the Smiley system. In the bind method, if remoteTrack = zero, score = 0F is written. Everywhere except Anilist with the Smiley system 0 is included in the ScoreList and accordingly selections.indexOf(selection) will return 0 if no score is set. Also 0 in all these systems is not an score and means no score.

Reply:
That's a good call, but the fact that it won't handle the smiley system is also a problem.

Reply:
Sure. Can you just update it with the takeIf then?

Reply:
Of course, as soon as I get to the computer